dist/images/ovnkube.sh

```diff
@@ -847,7 +847,7 @@ ovn-node() {
   wait_for_event process_ready ovn-controller
 
   hybrid_overlay_flags=
-  if [[ -n "${ovn_hybrid_overlay_enable}" ]]; then
+  if [[ ${ovn_hybrid_overlay_enable} == "true" ]]; then
```

Comment:
The master node logic is as follows:
```
  if [[ ${ovn_hybrid_overlay_enable} == "true" ]]; then
    hybrid_overlay_flags="--enable-hybrid-overlay"
    if [[ -n "${ovn_hybrid_overlay_net_cidr}" ]]; then
      hybrid_overlay_flags="${hybrid_overlay_flags} --hybrid-overlay-cluster-subnets=${ovn_hybrid_overlay_net_cidr}"
    fi
  fi
```
Should the node code be processing the CIDR as well?

Reply:
@Billy99 yes our node code must get the CIDR as well, or some things won't work like the Windows hybrid overlay route on the management interface

Reply:
Thanks @dcbw. Updated to include the CIDR processing on node, same as master.